test/testBearingRange2D.jl

```diff
@@ -122,23 +122,23 @@ res = testFactorResidualBinary( p2br,
 @test norm(res) < 1e-14
 
 ##
-#TODO Update to new CalcFactor 
-# test parametric Pose2Point2BearingRange
-f = Pose2Point2BearingRange(Normal(0.0,1), Normal(10.0,1))
-@test isapprox(f([0.,0,0], [10.,0]), 0, atol = 1e-9)
-@test isapprox(f([0,0,pi/2], [0.,10]), 0, atol = 1e-9)
-
-f = Pose2Point2BearingRange(Normal(pi/2,1), Normal(10.0,1))
-@test isapprox(f([0.,0,0], [0.,10]), 0, atol = 1e-9)
-@test isapprox(f([0,0,pi/2], [-10.,0]), 0, atol = 1e-9)
-
-f = Pose2Point2BearingRange(Normal(pi,1), Normal(10.0,1))
-@test isapprox(f([0.,0,0], [-10.,0]), 0, atol = 1e-9)
-@test isapprox(f([0,0,pi/2], [0.,-10]), 0, atol = 1e-9)
-
-f = Pose2Point2BearingRange(Normal(-pi/2,1), Normal(10.0,1))
-@test isapprox(f([0.,0,0], [0.,-10]), 0, atol = 1e-9)
-@test isapprox(f([0,0,pi/2], [10.,0]), 0, atol = 1e-9)
+# #TODO Update to new CalcFactor 
```

Comment:
yeah, i've been trying to find a nice API in IIf for these tests.  I basically start with IIF.testResidualBinary (or near about)